subsys/partition_manager/pm.yml.zboss

```diff
@@ -15,7 +15,7 @@ zboss_product_config:
   placement:
     after: [zboss_nvram]
 #ifdef CONFIG_BUILD_WITH_TFM
-    align: {start: CONFIG_NRF_SPU_FLASH_REGION_SIZE}
+    align: {start: CONFIG_NRF_TRUSTZONE_FLASH_REGION_ALIGNMENT}
```

Comment:
With _SIZE replacing the _ALIGNMENT in the end?

```suggestion
    align: {start: CONFIG_NRF_TRUSTZONE_FLASH_REGION_ALIGNMENT}
```

Reply:
Good catch, thanks!